client/src/lib/solana/solid-data.ts

```diff
@@ -77,8 +99,11 @@ export class SolidData extends Assignable {
 
   static empty(): SolidData {
     return new SolidData({
+      cluster: ClusterType.mainnetBeta(),
+      // account: SolidPublicKey.fromPublicKey(new Account().publicKey),
+      authority: SolidPublicKey.fromPublicKey(new Account().publicKey),
```

Comment:
Keep in mind that `new Account()` is actually a new random key!

Reply:
For empty() I think we want this though, no? It is basically just a dummy DID. Once i do step 3 in the [plan](https://civicteam.atlassian.net/wiki/spaces/ID/pages/1858076717/Optimised+Storage+for+SOLID+DIDs), we will be able to have authorityless DIDs , so then I can just make this null here.

I should remove that comment on line 103 though :)